pkg/controller/ads/ads_processor.go

```diff
@@ -32,12 +32,14 @@ import (
 
 	admin_v2 "kmesh.net/kmesh/api/v2/admin"
 	core_v2 "kmesh.net/kmesh/api/v2/core"
+	"kmesh.net/kmesh/pkg/constants"
 	"kmesh.net/kmesh/pkg/controller/config"
 	"kmesh.net/kmesh/pkg/utils/hash"
 )
 
 const (
 	apiVersionInfo = "v2"
+	mode           = constants.AdsMode
```

Comment:
Since we already have constant.AdsMode as a constant, why define a constant here？

Reply:
+1

Reply:
fixed